Tests/js/views/assets/ez-universaldiscoveryview-tests.js

```diff
@@ -342,6 +343,10 @@ YUI.add('ez-universaldiscoveryview-tests', function (Y) {
                 this.method.get('multiple'),
                 "The method multiple flag should be resetted to false"
             );
+            Assert.isUndefined(
```

Comment:
undefined ? I guess it's because https://github.com/ezsystems/PlatformUIBundle/pull/590/files#r63483229  :)